src/styles/components/_form-text.scss

```diff
@@ -286,27 +500,51 @@ input[type='search']::-moz-search-clear-button {
   }
 }
 
+// Estilos visibles para validación
+.is-valid ~ .valid-feedback,
+.is-invalid ~ .invalid-feedback {
+  display: flex !important;
+  justify-content: start;
+  align-items: center;
+  padding: 8px 12px;
+  max-width: 443px;
+  border-radius: 8px;
+  font-size: 14px;
+  margin-top: 8px;
+  color: $body-color;
+  position: relative;
+  padding-left: 44px; // Espacio para el ícono a la izq.
```

Comment:
este padding-left se pisa con el de la clase is-invalid ~ .invalid-feedback (linea 525)